docs/python/editing.md

```diff
@@ -86,20 +86,38 @@ While editing, you can right-click different identifiers to take advantage of se
-![Adding an import](images/editing/quickFix.gif)
+
+![Add import code action](images/editing/quickFix.png)
 
 This Code Action also recognizes some of the popular abbreviations for the following common Python packages: `numpy` as np, `tensorflow` as tf, `pandas` as pd, `matplotlib.pyplot` as plt, `matplotlib` as mpl, `math` as m, `scipi.io` as spio, and `scipy` as sp, `panel` as pn, and `holoviews` as hv.
 
 ![Common package abbreviations](images/editing/packageAbbreviations.gif)
 
-The import suggestions list is ordered with import statements for packages (or modules) at the top. It will also include statements for more modules and/or members (classes, objects, etc.) from specified packages.
+The import suggestions list displays  the top 3 high-confidence import options, prioritized based on: most recently used imports, symbols from the same module, symbols from the standard library, symbols from user modules, symbols from third-party packages, and finally sorting by module and symbol name.  In the case where the 3 high-confidence import options aren’t what you are looking for, Pylance has a **Search for additional import matches** Code Action, which displays a quick pick menu that allows you to search for import options that prefix-match the missing import symbol.
+
+![Search for additional import matches Code Action](images/editing/search-imports-code-action.gif)
+
+Pylance also offers a **Change spelling** Code Action, which displays import suggestions for missing imports due to typos.
+
+![Change spelling code action on missing import due to a typo](images/editing/change-spelling-code-action.gif)
+
+
+Just like with auto imports, only top-levels symbols are suggested by default. You can customize this behavior through the `python.analysis.packageIndexDepths` setting. Note that user symbol aliases (e.g. `import foo as bar`) are not supported by these Code Actions.
+
+### Implement All Inherited Abstract Classes
 
-Just like with auto imports, only top-levels symbols are suggested by default. You can customize this behavior through the `python.analysis.packageIndexDepths` setting.
+In Python, abstract classes serve as "blueprints" for other classes and help build modular, reusable code by promoting clear structure and requirements for subclasses to adhere to. To define an abstract class in Python, you can create a class that inherits from the `ABC` class in the `abc` module, and annotate its methods with the `@abstractmethod` decorator. Then, you can create new classes that inherit from this abstract class, and define an implementation for the base methods.
```

Comment:
I added this code action as refactoring since my understanding is that quick fix is to get rid of the diagnostics. [Another enhancement](https://github.com/microsoft/pylance-release/issues/5757) that Heejae did to fix the `reportAbstractUsage` diagnostic is a quick fix.

Reply:
very good point, thanks for bringing this up! I moved this to the refactoring section, and once the quick fix lands on stable I can add it back to the quick fix section :)